GroupChat/src/main/webapp/index.html

```diff
@@ -0,0 +1,135 @@
+
+        <script src="resource/js/underscore-min.js" type="text/javascript"></script>
+        <script src="resource/js/backbone-min.js" type="text/javascript"></script>
+        <script src="resource/js/backbone.localStorage-min.js" type="text/javascript"></script> 
+
+
+        <link href="./resource/css/bootstrap.css" rel="stylesheet">
+        <link href="./resource/css/index.css" rel="stylesheet">
+        <link href="./resource/css/bootstrap.css" rel="stylesheet">
+
+        <!-- Templates -->
+        <script type="text/template" id="message-template">
+            <td class="received"> <%- received %> </td>
+            <td class="user label label-info"> <%- sender %> </td>
+            <td class="message badge" id="message<%- message %>"> <%- message %> </td>
+        </script>  
+
+        <script type="text/template" id="app-template">
+            <div class="container chat-signin">
+            <form class="form-signin">
+                <h2 class="form-signin-heading">Chat sign in</h2>
+                <label for="nickname">Nickname</label> <input type="text"
+                                                              class="input-block-level" placeholder="Nickname" id="nickname">
+
+                <button class="btn btn-large btn-primary" type="submit"
+                        id="enterRoom">Sign in</button>
+            </form>
+        </div>
+            <div class="container chat-wrapper">
+               <h2 class="alert alert-success"></h2>
+
+                <section id="main">
+                    <table id="response" class="table table-bordered"></table>
+                </section>
+
+                <table id="response" class="table table-bordered"></table>
+                <fieldset>
+                    <legend>Enter your message..</legend>
+                    <div class="controls">
+                        <input type="text" class="input-block-level" placeholder="Your message..." id="message" style="height:60px"/>
+                        <input type="button" class="btn btn-large btn-block btn-primary"
+                               value="Send message" id="buttonMessage"/>
+                        <button class="btn btn-large btn-block" type="button" id="leave-room">Leave
+                            room</button>
+                    </div>
+                </fieldset>
+            </div>
+        </script>
+
+
+
+
+        <script src="resource/js/index.js"></script>
+
+    </head>
+
+    <body>
+        <section id="messageApp">
+
+
+        </section>
+
+    </body>
+         <script type="text/javascript">
+    app.AppView = Backbone.View.extend({
+    template: _.template($('#app-template').html()),
+    render: function () {
+        this.$el.html(this.template());
+    },
+    initialize: function () {
+        this.render();
+      
+        this.input = this.$('#message');
+        console.log(this.input);
+        app.MessageList.on('add', this.addOne, this);
```

Comment:
its better to use Backbonbe.View listenTo method to add event listeners to view methods